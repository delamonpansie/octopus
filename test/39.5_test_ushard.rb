#!/usr/bin/ruby
# coding: utf-8

$: << File.dirname($0)
require '39_test_ushard'


$one_env.meta 'shard 1 create por'

# change shard type
log_try { $one_env.meta 'shard 1 type raft' }


$one_env.meta 'shard 1 add_replica two'
$one_env.meta 'shard 1 add_replica three'

$one_env.meta 'shard 1 type raft'

sleep 5

$one_env.meta 'shard 1 obj_space 0 create tree unique string 0'
$one.insert ["aaa"], :shard => 1
