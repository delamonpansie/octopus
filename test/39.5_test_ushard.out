# one.ping()
:pong

# two.ping()
:pong

# one.meta(shard 1 create por)
ok

# one.meta(shard 1 type raft)
error: 0x202 bad peer count

# one.meta(shard 1 add_replica two)
ok

# one.meta(shard 1 add_replica three)
ok

# one.meta(shard 1 type raft)
ok

# one.meta(shard 1 obj_space 0 create tree unique string 0)
ok

# one.insert(["aaa"], {:shard=>1})
1

